.devcontainer/Dockerfile

```diff
@@ -20,11 +20,11 @@ RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
 
 # install PowerShell
 RUN echo INSTALL POWERSHELL \
-&& dotnet tool install --global PowerShell 
+&& dotnet tool install --global PowerShell --version 7.0.6
```

Comment:
This change was required because the latest version of PowerShell Core does not support .NET v3.1 and the Bot Framework hasn't fully moved to .NET v5 yet.